loom/common/platform/platformAndroidJni.cpp

```diff
@@ -294,7 +294,6 @@ const char *LoomJni::getWritablePath()
         "()Ljava/lang/String;"))
     {
         jstring str = (jstring)t.getEnv()->CallStaticObjectMethod(t.classID, t.methodID);
-        t.getEnv()->DeleteLocalRef(t.classID);
```

Comment:
It looks like according to JNI docs it would be cleaned up anyway. So this should be fine. Landing. See http://stackoverflow.com/questions/2093112/why-i-should-not-reuse-a-jclass-and-or-jmethodid-in-jni